build/common.project.props

```diff
@@ -124,7 +124,8 @@
     <MinimumVisualStudioVersion>15.0</MinimumVisualStudioVersion>
     <VsixOutputDirName>VS15</VsixOutputDirName>
     <CodeAnalysisRuleSet Condition="'$(CodeAnalysisRuleSet)' == ''">$(BuildCommonDirectory)NuGet.ruleset</CodeAnalysisRuleSet>
-    <EnforceCodeStyleInBuild Condition=" '$(EnforceCodeStyleInBuild)' != '' " >true</EnforceCodeStyleInBuild>
+    <EnforceCodeStyleInBuild Condition="'$(EnforceCodeStyleInBuild)' == ''">true</EnforceCodeStyleInBuild>
```

Comment:
If I am reading the [document](https://learn.microsoft.com/dotnet/fundamentals/code-analysis/overview?tabs=net-8#latest-updates) correctly, we should also set the `AnalysisLevel` MSBuild property to a specific value to lock the warnings to a particular version of SDK. Otherwise, we may get frequent build failures as we keep building on the preview versions of SDK.

Reply:
I like this, at least we can be intentional about taking in new rules as new versions of the SDK are released. Added.